arbitrum-docs/das/daserver-instructions.mdx

```diff
@@ -1,4 +1,10 @@
-# Data Availability Server Instructions
+---
+title: "Run a Data Availability Server"
```

Comment:
```suggestion
title: "How to run a Data Availability Server"
```

@symbolpunk can confirm here, but I think the convention was to use "How to ...." in the title of the page, while the sidebar would just mention the action.